scripts/irods/test/test_ifsck.py

```diff
@@ -83,3 +83,31 @@ def test_ifsck_continues_when_user_cannot_access_subdirectory__issue_5358(self):
             for d in dirs:
                 os.chmod(d, 0o777)
 
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, "Skip for Topology Testing, checks vault")
+    def test_ifsck_unregistered_file_detected__6467(self):
+        directory_name = 'test_ifsck_unregistered_file_detected__6467'
+        filename1 = 'test_ifsck_unregistered_file_detected__6467_f1'
+        filename2 = 'test_ifsck_unregistered_file_detected__6467_f2'
+        full_directory_path = '%s/%s' % (os.getcwd(), directory_name)
```

Comment:
If you want a more "automatic" cleanup, you can use `self.admin.local_session_dir` (or `self.user0` or whatever) which will get cleaned up with the session. Then you won't have to remove the files and the directory in the `finally` block.

Reply:
done